src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -961,9 +943,9 @@ FixedwingAttitudeControl::task_main()
 				float speed_body_v = 0.0f;
 				float speed_body_w = 0.0f;
 				if(_att.R_valid) 	{
```

Comment:
The _att.R_valid flag here is the only dependency left from vehicle_attitude topic. As far as I see the flag is always set to true anyway, so we can take away the check and unsubscribe from vehicle_attitude, as we directly calculate R locally from the quaternions.

I was unsure wether I see the real reason behind the flag so I left it there for now..should it go away?